compiler/passes/buildDefaultFunctions.cpp

```diff
@@ -1999,6 +2040,11 @@ static void buildDefaultReadWriteFunctions(AggregateType* ct) {
 
     normalize(fn);
   }
+
+  if (ct->builtReaderInit && AnySerialize && readerInit != nullptr) {
+    auto msg = new_StringSymbol("Initializers called by IO for deserialization are not compiler-generated when a user-defined 'serialize' or 'deserialize' method exists");
```

Comment:
Have we landed on "deserializing initializers" as the term? I think that might sound better here.

Reply:
I think that's a good term and one I plan to use in documentation after introducing it, but I think this text is a bit clearer.